airbyte-integrations/connectors/destination-elasticsearch-strict-encrypt/src/main/java/io/airbyte/integrations/destination/elasticsearch/ElasticsearchStrictEncryptDestination.java

```diff
@@ -38,4 +46,25 @@ public ConnectorSpecification modifySpec(ConnectorSpecification originalSpec) th
     return spec;
   }
 
+  @Override
+  public AirbyteConnectionStatus check(JsonNode config) throws Exception {
+
+    final ConnectorConfiguration configObject = convertConfig(config);
+    if (Objects.isNull(configObject.getEndpoint())) {
+      return new AirbyteConnectionStatus()
+          .withStatus(AirbyteConnectionStatus.Status.FAILED).withMessage("endpoint must not be empty");
```

Comment:
Same as below but can we have this `withStatus` and `withMessage` on two separate lines to follow how other usages of `AirbyteConnectionStatus` settings are set?

Also minor nit but maybe instead of `must not be empty` to have something like `Server Endpoint is a required field`

Reply:
Updated, thanks